### PR TITLE
[WIP]Translation support

### PR DESCRIPTION
**Pull request by stijnhau**
These are some commits to at translation support + dutch translations

watch out the first 4 ore changes form you repositoy merges with my fork.

### CODE REVIEW

**Comment by ojhaujjwal** [comment]
I have not used translator because I did not need it .So I am pretty new to this translator thing.

File: language/de_DE.po [hunk 1]
```diff
@@ -0,0 +1,120 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: ZfcUser\n"
```
Comment (ojhaujjwal) [inline language/de_DE.po:3]: What`s this?

File: language/de_DE.po [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+#~ msgid "Verify New Email"
+#~ msgstr "E-Mail wiederholen"
+
+#~ msgid "Display Name"
+#~ msgstr "Anzeigename"
+
+#~ msgid "Password"
+#~ msgstr "Passwort"
+
+#~ msgid "Password Verify"
```
Comment (ojhaujjwal) [inline language/de_DE.po:61]: Wait this module does not contain this! It is in ZfcUser! Why do you need this?

File: language/de_DE.po [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+#~ msgid "New Password"
+#~ msgstr "Neues Passwort"
+
+#~ msgid "Please type the following text"
+#~ msgstr "Geben Sie bitte den folgenden Text ein"
+
+#~ msgid "Submit"
+#~ msgstr "Senden"
+
+#~ msgid "Sign In"
```
Comment (ojhaujjwal) [inline language/de_DE.po:79]: this module does not contain this! It is in ZfcUser! Why do you need this?

Comment (stijnhau) [inline language/de_DE.po:79]: Forgotten to remove the unneed translation(purge ... in poedit)

File: language/de_DE.po (hunk 1 above)
Comment (stijnhau) [inline language/de_DE.po:3]: i reused the files from ZfcUser and added with poedit the new strings to translate in it.

**Comment by ojhaujjwal** [comment]
@stijnhau Please add [WIP] at the start of the title of the issue until it`s complete!

**Comment by stijnhau** [comment]
Also i found out i forgot to add 1 string.

**Comment by ojhaujjwal** [comment]
@stijnhau You closed it for now or for ever?

**Comment by stijnhau** [comment]
Going to make a new one because of the commits that came from you version in this pull request.

**Comment by stijnhau** [comment]
New pull request has been made

**Pull request closed without merging**